### ISSUE TITLE
Toggle controls and toggle playlist in fullscreen mode

### ISSUE THREAD

**Issue by kentherzz** [celluloid-player/celluloid#456]
Enhancement Request:

This player is starting to get really perfect! I have a suggestion to make things more tidy:

In fullscreen mode it would be nice to have the same features as we have in windowed mode. I really like to hide the controls as I'm watching IPTV streams. In windowed mode this works perfect but in fullscreen mode the control bar is down there. As a bonus I would also suggest that it should be possible to toggle the playlist in fullsceen mode as well. This can make it easier for a user to select channels. It can then dissapear in some seconds and only show up on mouse activity maybe. 

I really hope you will concider these two things as it will make this player a perfect companion for those who use a HTPC with keyboard and mouse.

Thanks for your time!

**Comment by gnome-mpv** [comment]
>I really like to hide the controls as I'm watching IPTV streams. In windowed mode this works perfect but in fullscreen mode the control bar is down there.

Currently, the controls should automatically hide if the cursor stops moving for a while. Do you mean that there should be an option for it to always remain hidden?

**Comment by kentherzz** [comment]
Yes please, there are usually no need for controls when playing IPTV streams.
Can you make this an option @gnome-mpv ?

**Comment by gnome-mpv** [comment]
Yes, I think that's a reasonable request.

**Comment by kentherzz** [comment]
@gnome-mpv Even the top bar could be hidden together with the status bar in fullscreen mode as a user could just double click anywhere on the player or ESC to return to windowed mode. I'm very used to this on mpv in fullscreen mode where nothing is hovering on the screen if I dont want to as OSD can be enabled/disabled. Thanks! I'm looking forward to the next version.

**Comment by gnome-mpv** [comment]
If you want to emulate this behavior right now, you can just set the dconf key `controls-unhide-cursor-speed` to a very large value, e.g. `dconf write /io/github/celluloid-player/celluloid/controls-unhide-cursor-speed 1.79769e+308`.

**Comment by kentherzz** [comment]
Nice tip! Where can I find that dconf key, I installed dconf-editor but cannot find controls-unhide-cursor-speed.

**Comment by gnome-mpv** [comment]
It should be under this path: `/io/github/celluloid-player/celluloid/`

**Comment by kentherzz** [comment]
Cannot find it. Maybe I didnt fully install it. Maybe a issue with Flatpak so I tried to build it myself... Is it possible for you to build version 0.17 for Ubuntu 19.04 amd64? I am not that experienced and maybe im doing something wrong when building it. Tried your guide.

**Comment by gnome-mpv** [comment]
Try this [package](https://github.com/celluloid-player/celluloid/files/3538075/celluloid_0.17-1_amd64.deb.gz). You'll need to run `gunzip celluloid_0.17-1_amd64.deb.gz` to decompress it first.

**Comment by kentherzz** [comment]
@gnome-mpv Thank you! :+1:
